### PR TITLE
Doc tracing

### PR DESCRIPTION
**Pull request by tybalex**
This is the documentation to manually setup the tracing capability in downstream cluster.

### CODE REVIEW

File: docs/installation/opni_agent/enable_tracing.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+Opni agent doesn't support the tracing capability yet. This guild will walk you through an example of using Opentelemetry Collector to ship trace data to Opni.
+
+## Prerequisites
+* The [Logging Backend](/docs/installation/opni/backends.md) is enabled in your Opni cluster.
+* Instrument your applications in the downstream cluster.
+* Cert-manager is installed in the downstream cluster.
+    <summary>Install cert-manager using kubectl apply with static manifests</summary>
```
Comment (sanjay920) [inline docs/installation/opni_agent/enable_tracing.md]: why put this in a `summary`?

File: docs/installation/opni_agent/enable_tracing.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+    ```bash
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml
+    ```
+
+## Getting Started
+**1. Install Data-Prepper**
+:::note
+if you already have Opni-Agent installed in your downstream cluster and have the logging capability enabled, skip this step.
+In this case,
```
Comment (sanjay920) [inline docs/installation/opni_agent/enable_tracing.md]: unfinished sentence?

Comment (sanjay920) [reply docs/installation/opni_agent/enable_tracing.md]: let's assume the user has enabled the logging capability. we will support traces without logs in the future but for now let's force logs+traces to be enabled at the same time

**Review by sanjay920 — CHANGES_REQUESTED** [review]

File: docs/installation/opni_agent/enable_tracing.md (hunk 2 above)
Comment (tybalex) [reply docs/installation/opni_agent/enable_tracing.md]: ok.

File: docs/installation/opni_agent/enable_tracing.md (hunk 1 above)
Comment (tybalex) [reply docs/installation/opni_agent/enable_tracing.md]: copy-pasted from somewhere else lol

**Comment by dbason** [comment]
Do we want to include documentation on how to turn on tracing in the demo application?

**Comment by tybalex** [comment]
> Do we want to include documentation on how to turn on tracing in the demo application?

you mean the instrumentation?

**Comment by dbason** [comment]
> you mean the instrumentation?

Yup

**Review by sanjay920 — APPROVED** [review]

**Comment by sanjay920** [comment]
@tybalex will update docs further in [this issue](https://github.com/rancher/opni/issues/899)

**Comment by tybalex** [comment]
> > you mean the instrumentation?
> 
> Yup

let me do that in a separate PR

**Pull request merged**